modules/feature-management/src/main/java/org/mule/runtime/feature/internal/togglz/activation/MuleTogglzActivationStrategyProvider.java

```diff
@@ -32,19 +39,23 @@ public static ActivationStrategyProvider getDefaultActivationStrategyProvider()
 
   @Override
   public List<ActivationStrategy> getActivationStrategies() {
-    addSpiActivationStrategies();
+    addTogglzActivationStrategies();
     addMuleActivationStrategies();
 
     return strategies;
   }
 
-  private void addMuleActivationStrategies() {
-    strategies.add(MuleTogglzActivatedIfEnabledActivationStrategy.getInstance());
+  private void addTogglzActivationStrategies() {
+    strategies.add(new UsernameActivationStrategy());
+    strategies.add(new GradualActivationStrategy());
+    strategies.add(new ScriptEngineActivationStrategy());
+    strategies.add(new ReleaseDateActivationStrategy());
+    strategies.add(new ServerIpActivationStrategy());
+    strategies.add(new UserRoleActivationStrategy());
+    strategies.add(new SystemPropertyActivationStrategy());
   }
 
-  private void addSpiActivationStrategies() {
-    for (ActivationStrategy activationStrategy : ServiceLoader.load(ActivationStrategy.class)) {
```

Comment:
In the modular world, a module cannot load service impelementations from an interface declared in another module.